packages/headless/src/controllers/core/facets/range-facet/numeric-facet/headless-core-numeric-facet.ts

```diff
@@ -183,17 +206,33 @@ export function buildCoreNumericFacet(
       this.toggleSelect(selection);
     },
 
+    enable() {
+      dispatch(enableFacet(facetId));
+    },
+
+    disable() {
+      dispatch(disableFacet(facetId));
+    },
+
     get state() {
-      return rangeFacet.state;
+      const enabled = getIsEnabled();
+
+      return {
+        ...rangeFacet.state,
+        enabled,
```

Comment:
could it be directly in the rangeFacet state here?

Reply:
woops, looks like I copy and pasted too fast. Yes, it definitely should and I'll fix it.